### DISCUSSION TITLE
Is there any plan to add unit testing capabilities to the SQL Database Project?

### DISCUSSION THREAD

**Discussion by freddie2025** [microsoft/azuredatastudio#23955] [Q&A] [answered]
In the SSDT project, we had the ability to unit test our database code using the following library: Microsoft.Data.Tools.Schema.Sql.UnitTesting. Is there any plan to offer a .net core equivalent?

**Comment by dzsquared** [comment, marked as answer]
There isn't an immediate plan for this, but it is on our radar: https://github.com/microsoft/azuredatastudio/issues/12975

In addition to the .NET SQL unit tests, the community library tSQLt is a good option for unit testing against SQL projects. (deploy project and unit tests to container in pipeline, run tests)
